demo/switch/switchLatest.php.expect

```diff
@@ -0,0 +1,6 @@
+Next value: 0
+Next value: 1
+Next value: 2
+Next value: 3
+Next value: 4
+Complete!
```

Comment:
@asm89 I updated the expected output to the correct info.  You can see how it doesn't work with the Immediate Scheduler.  https://travis-ci.org/ReactiveX/RxPHP/jobs/111093463#L236

Once https://github.com/ReactiveX/RxPHP/issues/32 is resolved, this should work.